templates/vdc/vdc.py

```diff
@@ -146,15 +150,21 @@ def uninstall(self):
         '''
         Delete VDC
         '''
+        self.state.check('actions', 'install', 'ok')
```

Comment:
This line would throw an exception if the install action state was not set to ok.
Should we do this in an uninstall/delete method or exit without error/exception and log that the install state was not ok?
@FastGeert?

Reply:
Good catch @chrisvdg, when deleting, we should not care if the install succeeded.